### PR TITLE
🚧 collections #3

### PR DESCRIPTION
**Pull request by herrmannplatz**
Fixes WEBAPP-3496/3505/3515/3525

### CODE REVIEW

**Comment by gregor** [comment]
I think we are still missing CSS for some cursors on Windows though.

**Comment by herrmannplatz** [comment]
I added it for the preview images. Where else?

**Comment by Yserz** [comment]
Would be nice to have timestamps in 24h format just like for the message timestamps.

File: app/script/view_model/content/CollectionViewModel.coffee [hunk 1]
```diff
@@ -42,10 +47,9 @@ class z.ViewModel.content.CollectionViewModel
     @conversation_et conversation_et
     @conversation_repository.get_events_for_category conversation_et, z.message.MessageCategory.LINK_PREVIEW
     .then (message_ets) =>
-      if message_ets.length is 0
+      @populate_items message_ets
+      if @images().length is 0 and @files().length is 0 and @links().length is 0
```
Comment (lipis) [inline app/script/view_model/content/CollectionViewModel.coffee]: maybe?
```coffeescript
if @images().length + @files().length + @links().length is 0
```

File: app/style/content/conversation/conversation-input.less [hunk 2]
```diff
@@ -116,7 +116,7 @@
   border-width: 1px;
   border-style: solid;
   font-size: @font-size-xs;
-  font-weight: 600;
+  font-weight: @font-weight-bold;
   line-height: @avatar-diameter-xs - 2;
```
Comment (lipis) [inline app/style/content/conversation/conversation-input.less:120]: not part of this PR.. but maybe ` - 2px`?
